base_layer/core/src/validation/validation_utils.rs

```diff
@@ -0,0 +1,110 @@
+// Copyright 2019. The Tari Project
```

Comment:
call this file `helpers.rs` or even `functions.rs` maybe?
Reasons:
* it's already in the validation namespace (no need for the `validation_` prefix)
* `utils` makes me think of Swiss Army knife / very general functions.

Reply:
Renamed to `helpers.rs`.